wger/settings_global.py

```diff
@@ -91,12 +91,12 @@
 # added list of external libraries to be installed by bower
 BOWER_INSTALLED_APPS = (
     'jquery#2.1.x',
-    'bootstrap',
+    'bootstrap#3.3.5',
     'd3',
     'shariff',
```

Comment:
use shariff#1.24.1 which is the one that has build folder

Reply:
Okay thank you, will work on that